### PR TITLE
Update CHANGELOG and version in prep for 1.18.0 release

### PR DESCRIPTION
**Pull request by witriew**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## [1.18.0] - 2020-03-30
+### Added
+ * relay: Include time since last activity in dropped call log (#774)
+ * connection: Separate out lastActivity into recv and send activity (#770)
+ * Add options to allow overriding SendBufferSize pre process name prefix (#772)
```
Comment (prashantv) [inline CHANGELOG.md]: s/pre/per

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## [1.18.0] - 2020-03-30
+### Added
+ * relay: Include time since last activity in dropped call log (#774)
+ * connection: Separate out lastActivity into recv and send activity (#770)
```
Comment (prashantv) [inline CHANGELOG.md]: maybe call out that introspection tracks last activity separately for read/write (want to think about user impact in the changelog)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## [1.18.0] - 2020-03-30
+### Added
+ * relay: Include time since last activity in dropped call log (#774)
```
Comment (prashantv) [inline CHANGELOG.md]: here and below, relay changes can be dropped since this is intended for users

File: CHANGELOG.md (hunk 2 above)
Comment (witriew) [reply CHANGELOG.md]: Good call. Thanks!

**Pull request merged**